pyproject.toml

```diff
@@ -29,6 +29,7 @@ impulse = "^1.0"
 pylint = "^2.16.2"
 pytest = "^7.2.2"
 setuptools = "^67.5.1"
+elasticai-runtime-env5 = {git = "https://github.com/es-ude/elastic-ai.runtime.enV5.git", rev = "develop"}
```

Comment:
Do we want to switch runtime version automatically as it gets updated or do we want to switch manually by changing the commit hash or version tag here? The later case seems more safe to me.

Reply:
On one hand I agree with you. On the other I think it would make sense to automatically update to develop revision in the runtime, so we can test their newest features.

Reply:
Fixed the git revision.